app/scripts/controllers/authentication/authentication-controller.ts

```diff
@@ -152,6 +157,10 @@ export default class AuthenticationController extends BaseController<
     return profile;
   }
 
+  public isSignedIn(): boolean {
+    return this.state.isSignedIn;
+  }
```

Comment:
This is duplicate from an earlier PR. Will be removed once User Storage PR is merged.